.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
it's not a problem, but you don't need to commit this file. If you want to, you can avoid committing it in future by adding it to a special file called `.gitignore`. See this guide here for how t do that https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Thank Ellie, I noted this and I will add the file into gitignore.